js/pages/questionnaire.js

```diff
@@ -359,17 +359,17 @@ function externalListeners(){
 
 //BUILDING SOCCER
 function buildHTML(soccerResults, question) {
-    
-    let responseElement = question.querySelector("div[class='response']");
+    let fieldset = question.querySelector('fieldset');
+    let responseElement = fieldset.querySelector("div[class='response']");
     
     if (responseElement) {
       let tmp = responseElement.cloneNode(false);
-      question.replaceChild(tmp, responseElement);
+      fieldset.replaceChild(tmp, responseElement);
       responseElement = tmp;
     } else {
       responseElement = document.createElement("div");
       responseElement.classList.add("response");
-      question.insertBefore(responseElement, question.childNodes[0]);
+      fieldset.insertBefore(responseElement, fieldset.childNodes[0]);
```

Comment:
`fieldset.prepend(responseElement)` should also work.

Reply:
Updated above, thanks.